src/data/pages/founding-members.js

```diff
@@ -171,10 +172,174 @@ export const fullData = [
+c+lGPJl4k0vETjHu8p8D9dd/O88qJkYSJrDD95cDUBU6HcNJxn6Zrm8JPhFY5MqQ
+ni5KV78C9vbjPsKCM9pUZwyAttLxw9gWJJdd95bX8nnuwrjqfcH77RFUFgZeOUI9
+8aoN88/Yj5F0o45Nlr/EuV0donvLc7TAY0kFIMEjV/J+1/O/Q+PXgL+ppsz4T0Bu
+VrCrPiUpZawkam1JQfKoFhB7woF70ivtqrSWEL2tkB5hNrQWACjSgNjgUQARAQAB
+tDtKb3lzdHJlYW0gRm91bmRpbmcgTWVtYmVycyA8Zm91bmRpbmcubWVtYmVyc0Bq
+c2dlbmVzaXMuY29tPokCVAQTAQgAPhYhBIbPXy+EULE6VXRw4yMvfOCfjcrpBQJg
+EoHNAhsDBQkDnX0ABQsJCAcCBhUKCQgLAgQWAgMBAh4BAheAAAoJECMvfOCfjcrp
+0gYP/1QcSWWNKlVIRLnZLV6a09KZn4wmBh2bZnWx+CQubWoysmdv03n3oZUjmpn/
+PUC1jo+xUzsgTKiweY+dRagsVsXf8Xu5lIeFCNfZjHhzkX15Jd/aQkmzGWblgPy7
+gFhsMzZLAwF330OnM/MKtk6sqgn4013hk55C9458SIJU4Mcf0Q8/2Tv8jL9XDbk0
+urO50lG6dcwJNCZTFN4agLA4gIzr2+X2EkntU9IKq+QnnfgdEhFEH19VBHJBedl4
+MYVxjKO/jegm0sNeYA6WZD4GdkaM4W6RAgf2g/3dwD+qSVvUo8O9EFUug237Nit6
+zskk8Jp3QRaOIZLwYZ6G/CL8lzCoBnlHIPXN5Jyb1fGghoJFP9jenb+ZZaQG/5sd
+jTid5bnnymuAm7Xq+7ZEx9nsKw63cOhBbyYBJjKQYWr8Aa5wAqzZpB84tjHT4X3h
+Te82315fao6uAdAFqxXQk1yI7+bdmmwlztyg7G03kybVTwKeoChzWcOxEsJbFUEP
+h7c2vCP33dD/+ewTnxkK4qpEXw9AlLWXvDhYOI+HMZDcM3LI9/hG+xXkLP0/APu3
+UbeHjB2GzikVnQeEWlbMGz0tMQO3wMaca0HBhJdw2dEd1pMME9Seb5cM65KyUndD
+Ah0v+BIu2nG2W4P5yf8lMbn2JipCbXFo7iQGQwLbLFIvIvKWuQINBGASgc0BEACl
+3kuOkRnWedcxL2dojKIMm3Bx86xB2jt27y+6kM52wGksb/5RibayiKqZG75HKKut
+ahjH2SIWZNBYnvEZHEUAusK6m1B1Nq2J3MxueO46E1FIykFRTi1vZ/nF3aIY6/r7
+eIHN4GHgdPyPzyidQvvHjNlP35USQ0dDSXczGy8pL8HtTt+S6dXxIn4IkZwjdOr7
+zC+eoWywvj1jrXIQOX+DvF7q5VRnr8/bmIAatFWX99YODtwauCOxTh1oiLs0Y+aN
+nlZWLxFu8KhNOqCMnGa+X73a3meCZ40kUeVnMXe2fgFHn5sV/67c/xhbMagKGsW7
+w2TNyHXMODZKSsvT/0uXcmFt5LELqO70JOaSEhqEUu4tlySz2jyj6MIrwGQK2blW
+Jker6G6KsnwdmBCIuJ6NDbt3JtergotMv5B6WKNcjH9ARNxUFF42hfKhH1/Zam2c
+dv7lFULRfOmuAHQ7BTqcVjkP8xrHDwY/3QjK8RePAQo5ZH5MAmHQ+cRAATpE5h0x
+g3A1kc5tBC7fZ7YVHev9aPJvEuTQN0Ruq/EKErEsaV+jFOFugzU9CaXhBgNr+E59
+bA5eOp13NOfgYRZ8WNEuythjBy3jsbRD0M6snb7ptiOwMu7zHSMfR9V2/XTauOXi
+ItrQ471QSscESsrE1LvR3pl6PqHazpUVABgaGOv4SwARAQABiQI8BBgBCAAmFiEE
+hs9fL4RQsTpVdHDjIy984J+NyukFAmASgc0CGwwFCQOdfQAACgkQIy984J+Nyuk3
+9BAAs/2a9WZNeG43ND3Esma1PSzQSDzn528cuc4s2OS6YJB6RY8YinU9VFXR4b+C
+NInuAGmqrlzeeT27f4kZTaacvRzdD0N5Vif1CkBD1acgoGf1S1NIcC2hPDuh3Cim
+19DzdHGv5u2aRNtkxoE249Vz8i+03nRtsDV+kKtXip4O60b4ICc6EPHPHRCjQiw4
+v6g80gZ7NWmDEaxK51aso41k4fMUeqRgFRjKo2Xp5tpK+FLCM3twBgzJVHV/vL/0
+YErscr5tJTnCmNpmYxIQ/O1kZg7elNarWrUmslfgSplhKMVjrm9B5C2CpxBUKkov
+9x7gYDO/9Jd3Rxe2hVdG/3Jy3law0z4r3nLFck33qHcrn3egtly18mGvJSrTn+Og
+pPbbOTPiQhM97A4ETL+xs8mEZnT8KUP83rta+iAPnE/OJfeY2CqE0A31bg0q3BO2
+oAHLwC1Q/4z7VvizkWq2qhBvdI9CKYCKpNfruR0jwAYSNKT/92PlRyU1+ganDW3Z
+ONBb4+CC/fKxyj7CmvVYDmL753AeVSWnzvcNuvLebrZCa8H10AIbQTfAG//hqoX5
+ZLae0VoRnU2EPG1Uh/W34+hXM46Zdk8xZt1Zl5Gkdk/1lb69Mj3RuPM9P0pFjS3N
+cqtfSrRMTi4uP5xa8b19Asnu0fkSAXnchOfW4M4GICuvgeo=
+=sbij
+-----END PGP PUBLIC KEY BLOCK-----`;
+
+export const JoystreamWSProvider = 'wss://rome-rpc-endpoint.joystream.org:9944/';
+
+export const foundingMembersJson =
+  'https://raw.githubusercontent.com/bwhm/founding-members/test-schema/data/scoring-example.json';
```

Comment:
```suggestion
  'https://raw.githubusercontent.com/Joystream/founding-members/main/data/fm-info.json';
```

As this should be the url once [this](https://github.com/Joystream/founding-members/pull/3) is merged...